### PR TITLE
kube/minikube: fix Sanskar Bhushan copyright statements

### PR DESCRIPTION
**Pull request by gctucker**
Fix the copyright statements for Sanskar Bhushan to follow the convention of listing the copyright owners directly rather than referring to the "authors".  In many cases, the authors don't own the copyright if they're writing the code as part of their work with an employer.

### CODE REVIEW

**Review by nuclearcat — APPROVED** [review]
LGTM

File: kube/minkube/init/init-pod.yaml [hunk 1]
```diff
@@ -16,9 +15,8 @@ spec:
     - name: data-volume
       mountPath: /data
     # resource limits needs to be discussed
-    resources: 
+    resources:
```
Comment (nuclearcat) [inline kube/minkube/init/init-pod.yaml:18]: I think it is not related to copyright fixes and can be separate commit?

Comment (gctucker) [reply kube/minkube/init/init-pod.yaml:18]: No that's just whitespace automatically removed.  It's normal for trailing whitespace to cause git noise.
